epregressions/runtests.py

```diff
@@ -351,17 +345,20 @@ def run_build(self, build_tree):
                     os.path.join(test_run_directory, 'HybridModel_Measurements_no_HVAC.csv')
                 )
 
-            if 'SolarShadingTest_Shading_Data.csv' in idf_text:
-                shutil.copy(
-                    os.path.join(build_tree['test_files_dir'], 'SolarShadingTest_Shading_Data.csv'),
-                    os.path.join(test_run_directory, 'SolarShadingTest_Shading_Data.csv')
-                )
-
-            if 'LocalEnvData.csv' in idf_text:
-                shutil.copy(
-                    os.path.join(build_tree['test_files_dir'], 'LocalEnvData.csv'),
-                    os.path.join(test_run_directory, 'LocalEnvData.csv')
-                )
+            # several checks that just bring a single file from the test files dir based on the filename as a keyword
+            single_file_checks = [
+                'HybridZoneModel_TemperatureData.csv',
+                'LookupTable.csv',
+                'SolarShadingTest_Shading_Data.csv',
+                'LocalEnvData.csv',
+                'SurfacePropGndSurfs.csv',
+            ]
+            for single_file_check in single_file_checks:
+                if single_file_check in idf_text:
+                    shutil.copy(
+                        os.path.join(build_tree['test_files_dir'], single_file_check),
+                        os.path.join(test_run_directory, single_file_check)
+                    )
```

Comment:
Collected all the similar checks of: if this filename is in the IDF, copy it from the test files directory.